scss/settings/_settings.scss

```diff
@@ -49,18 +49,30 @@
 $global-font-size: 100%;
 $global-width: rem-calc(1200);
 $global-lineheight: 1.5;
-$foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
-  success: #3adb76,
-  warning: #ffae00,
-  alert: #cc4b37,
+$foundation-colors: (
```

Comment:
`scss/settings/_settings.scss` is automatically generated. You do not have to update it.